packages/react-native-codegen/src/parsers/parsers-commons.js

```diff
@@ -817,6 +825,36 @@ function propertyNames(
     .filter(Boolean);
 }
 
+function extendsForProp(
+  prop: PropAST,
+  types: TypeDeclarationMap,
+  propName: 'argument' | 'expression',
+): ExtendedPropResult {
+  if (!prop[propName]) {
+    console.log('null', prop);
+  }
+
+  const name = prop[propName]?.id
+    ? prop[propName].id.name
+    : prop[propName]?.name;
```

Comment:
Similar concern as above, we should extract this in a `nameForArgument` function in the parsers.
It could be that the parsers already have this method implemented.